pkgs/tools/networking/openconnect/default.nix

```diff
@@ -10,27 +10,27 @@
 , zlib
 , vpnc-scripts
 , PCSC
-, head ? false
-  , fetchFromGitLab
-  , autoreconfHook
+, unstable ? false
```

Comment:
Can we please extract that into an unstable file and keep the shared stuff in a generic or common file?

Reply:
That's a good idea. I've reorganized everything in a default.nix/common.nix structure.